apps/www/pages/_app.tsx

```diff
@@ -14,6 +14,9 @@ import { post } from '~/lib/fetchWrapper'
 import { AuthProvider, ThemeProvider, useTelemetryProps } from 'common'
 import Head from 'next/head'
 
+import '@code-hike/mdx/dist/index.css'
```

Comment:
QQ, was this line required by code hike?

Reply:
No must have left it there by mistake.